docs/topics/braintree.rst

```diff
@@ -373,7 +373,7 @@ Webhook
 When Braintree completes certain actions, they will make a request to the
 configured webhook URL. That will be `payments-service <https://github.com/mozilla/payments-service/>`_
 which then passes it on to this endpoint. For more information see the
-`Braintree documentation <https://developers.braintreepayments.com/javascript+python/reference/general/webhooks>`_.
+`Braintree webhook documentation <https://developers.braintreepayments.com/javascript+python/reference/general/webhooks>`_.
```

Comment:
mmm, there is a `linkcheck` command in sphinx actually. We could add it to the tests.